x-pack/legacy/plugins/task_manager/lib/intervals.test.ts

```diff
@@ -5,15 +5,40 @@
  */
 
 import _ from 'lodash';
-import { assertValidInterval, intervalFromNow, minutesFromNow } from './intervals';
+import sinon from 'sinon';
+import {
+  assertValidInterval,
+  intervalFromNow,
+  intervalFromDate,
+  minutesFromNow,
+  minutesFromDate,
+  secondsFromNow,
+  secondsFromDate,
+} from './intervals';
+
+const mockedNow = new Date('2019-06-03T18:55:25.982Z');
+sinon.useFakeTimers(mockedNow.valueOf());
```

Comment:
So this may not actually be necessary, but I would prefer it if we do the `useFakeTimers` in a before and then reset them in an after.  That way we're not depending on the jest test isolation magic.